### PR TITLE
Update header text

### PR DESCRIPTION
**Pull request by SamueleA**
Updated header text following feedback to show instructions relative to currencies.

image

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
-        <Text color="text100">
-          This demo was created with&nbsp;
-          <Text color="text100" as="a" href={'https://sequence.build'} target="_blank" rel="noreferrer ">
-            Sequence Builder
+      {isConnected && !salesCurrencyIsLoading && (
+        <Box marginTop="5" marginBottom="4" style={{ maxWidth: 600 }}>
+          <Text color="text100">
+            This is a demo that showcases currency abstraction using Sequence services. Make sure there is some{' '}
+            {network?.nativeToken.symbol} in your connected wallet (
+            <Text color="text100" as="a" href={`${ETHERSCAN_URL}/address/${userAddress}`} target="_blank" rel="noreferrer ">
```
Comment (ScreamingHawk) [inline src/components/Header.tsx]: `"noreferrer "` has space.

File: src/components/Header.tsx [hunk 2]
```diff
@@ -1,22 +1,45 @@
...
+              {formattedUserAddress}
+            </Text>
+            ), then purchase one of the listed tokens with&nbsp;
```
Comment (ScreamingHawk) [inline src/components/Header.tsx:29]: Can we use `{' '}` or `&nbsp;` consistently?

File: src/utils/index.ts [hunk 3]
```diff
@@ -1,11 +1,9 @@
-export const truncateAtMiddle = (text: string, truncateAt: number) => {
-  let finalText = text
-
-  if (text.length >= truncateAt) {
-    finalText = text.slice(0, truncateAt / 2) + '...' + text.slice(text.length - truncateAt / 2, text.length)
+export const truncateAddress = (address: string, minPrefix: number = 20, minSuffix: number = 3): string => {
```
Comment (ScreamingHawk) [inline src/utils/index.ts:1]: No longer handles the case where address is `''`

**Review by ScreamingHawk — DISMISSED** [review]
Approved. Couple nits that aren't that important

**Comment by SamueleA** [comment]
@ScreamingHawk I've addressed the issues in the latest commit!

**Review by ScreamingHawk — APPROVED** [review]
LFG

**Pull request merged**